09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
+
+The code chunk below demonstrates this functionality:
+`linkGDAL()` searches the computer for a working GDAL installation and adds the location of the executable files to the PATH variable, allowing GDAL to be called on different operating systems.
+In the example below `ogrinfo` provides metadata above a vector dataset:
 
 ```{r, eval=FALSE, message=FALSE}
 link2GI::linkGDAL()
-cmd = paste("ogrinfo -ro -so -al", 
-            system.file("shape/nc.shp", package = "sf"))
+cmd = paste("ogrinfo -ro -so -al", system.file("shape/nc.shp", package = "sf"))
 system(cmd)
-#>INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
-#>    using driver `ESRI Shapefile' successful.
-#>
-#>Layer name: nc
-#>Metadata:
-#> DBF_DATE_LAST_UPDATE=2016-10-26
-#>Geometry: Polygon
-#>Feature Count: 100
-#>Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
-#>Layer SRS WKT:
-#>GEOGCS["GCS_North_American_1927",
-#>    DATUM["North_American_Datum_1927",
-#>        SPHEROID["Clarke_1866",6378206.4,294.9786982]],
-#>    PRIMEM["Greenwich",0],
-#>    UNIT["Degree",0.017453292519943295],
-#>    AUTHORITY["EPSG","4267"]]
-#>AREA: Real (24.15)
-#>PERIMETER: Real (24.15)
-#>CNTY_: Real (24.15)
-#>CNTY_ID: Real (24.15)
-#>NAME: String (80.0)
-#>FIPS: String (80.0)
-#>FIPSNO: Real (24.15)
-#>CRESS_ID: Integer (9.0)
-#>BIR74: Real (24.15)
-#>SID74: Real (24.15)
-#>NWBIR74: Real (24.15)
-#>BIR79: Real (24.15)
-#>SID79: Real (24.15)
-#>NWBIR79: Real (24.15)
-```
-
-This is of course a very simple example, and we could have achieved the same using `rgdal::ogrInfo()`.
-Still, it shows how we can use GDAL via the command-line from within R independently of other packages, and nothing stops us from using more advanced and/or not already implemented GDAL utilities in R.
+#> INFO: Open of `C:/Users/geocompr/Documents/R/win-library/3.5/sf/shape/nc.shp'
+#>     using driver `ESRI Shapefile' successful.
+#> 
+#> Layer name: nc
+#> Metadata:
+#>  DBF_DATE_LAST_UPDATE=2016-10-26
+#> Geometry: Polygon
+#> Feature Count: 100
+#> Extent: (-84.323853, 33.881992) - (-75.456978, 36.589649)
+#> Layer SRS WKT:
+#> GEOGCS["GCS_North_American_1927",
+#>     DATUM["North_American_Datum_1927",
+#>         SPHEROID["Clarke_1866",6378206.4,294.9786982]],
+#>     PRIMEM["Greenwich",0],
+#>     UNIT["Degree",0.017453292519943295],
+#>     AUTHORITY["EPSG","4267"]]
+#> AREA: Real (24.15)
+#> ...
+```
+
+This example --- which returns the same result as `rgdal::ogrInfo()` --- may be simple, but it shows how to use GDAL via the system command-line, independently of other packages.
+The 'link' to GDAL provided by **link2gi** could be used as a foundation for more doing more advanced GDAL work from the R or system CLI.^[
```

Comment:
not sure what you are referring to here. **link2GI** only searches your computer for a GDAL, GRASS, SAGA installation and adds the corresponding path to PATH, so that you can easily execute these programs from the system command line, but that's all

Reply:
I'm trying to create a direct replacement for the following sentence that was there before, which seemed dodgy to me (we should say how they should know how):

> In any case, you now have the tools to build an interface yourself.